src/styles/mixins/_animation.scss

```diff
@@ -0,0 +1,5 @@
+@mixin bdl-transition($time-length, $properties...) {
```

Comment:
should time-length be defaulted to the base value? Though then you would have to change args order.